tests/performance/parallel_algorithms/local/benchmark_is_heap_until.cpp

```diff
@@ -135,12 +139,9 @@ double run_is_heap_until_benchmark_par_unseq(int test_count,
 ///////////////////////////////////////////////////////////////////////////////
 int hpx_main(boost::program_options::variables_map& vm)
 {
-    unsigned int seed = (unsigned int)std::time(nullptr);
     if (vm.count("seed"))
         seed = vm["seed"].as<unsigned int>();
 
```

Comment:
Shouldn't `_rand` be seeded here again if `seed` is given on the command line? Same you've updated `benchmark_partition_copy.cpp`?

Reply:
I have updated the changes.Thanks for figuring out :)